src/lib/recipes.ts

```diff
@@ -422,6 +422,104 @@ const xiphosRecipes: IRecipe[] = [
   ),
 ];
 
+// T12 "Actaeon" Tankette vehicle recipes
+const actaeonTanketteRecipes: IRecipe[] = [
+  // Based on external data: armored car with tank armor, costs basic materials
+  createRecipe(
+    [{ stuff: Materials.ConstructionMaterials, count: 150 }],
+    [{ stuff: Vehicles.ActaeonTankette, count: 1 }]
+  ),
+  // Mass Production Factory variant
+  createRecipe(
+    [{ stuff: Materials.ConstructionMaterials, count: 450 }],
+    [{ stuff: Vehicles.ActaeonTankette, count: 3 }]
+  ),
+];
+
+// T5 "Percutio" vehicle recipes
+const percutioRecipes: IRecipe[] = [
+  // Anti-tank variant, costs refined materials
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 120 }],
+    [{ stuff: Vehicles.Percutio, count: 1 }]
+  ),
+  // Mass Production Factory variant
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 360 }],
+    [{ stuff: Vehicles.Percutio, count: 3 }]
+  ),
+];
+
+// T8 "Gemini" vehicle recipes
+const geminiRecipes: IRecipe[] = [
```

Comment:
Only one recipe:
10 x [Construction Materials](https://foxhole.wiki.gg/wiki/Construction_Materials) ConstructionMaterialsIcon.png
10 x [Assembly Materials I](https://foxhole.wiki.gg/wiki/Assembly_Materials_I) AssemblyMaterials1Icon.png
1x [T3 “Xiphos”](https://foxhole.wiki.gg/wiki/T3_%E2%80%9CXiphos%E2%80%9D) ArmoredCarVehicleIcon.png

Reply:
1 xipos is recipe requirements too